Kudu.Services/Diagnostics/ProfileManager.cs

```diff
@@ -18,16 +18,24 @@ internal static class ProfileManager
     {
         private const int ProcessExitTimeoutInSeconds = 180;
 
+        private const string SERVICE_PROFILER_AGENT_GUID = "F5091AA9-80DC-49FF-A7CF-BD1103FE149D";
+        private const string SERVICE_PROFILER_PROVIDER_GUID = "1C92BA2A-A990-480F-A02F-40068871CAAC";
+        private const string IIS_WWWSERVER_GUID = "3A2A4E84-4C21-4981-AE10-3FDA0D9B0F83";
+        private const string DIAGNOSTICS_HUB_AGENT_GUID = "4EA90761-2248-496C-B854-3C0399A591A4";
+
         private static ConcurrentDictionary<int, ProfileInfo> _profilingList = new ConcurrentDictionary<int, ProfileInfo>();
         private static object _lockObject = new object();
 
         // The profiling session timeout, this is temp fix before VS2015 Update 1.
         private static readonly TimeSpan _profilingTimeout = TimeSpan.FromMinutes(15);
+
+        private static readonly TimeSpan _profilingIISTimeout = TimeSpan.FromMinutes(3);
```

Comment:
Can you deploy a standard asp.net site or web api, on a small dedicated worker, enable IIS profiling and load test it with WCAT or some other http stress testing tool for 3 minutes and share the file? Also plz log the timestamp, I want to see how big the file would be and how the VM behaved during that time.
If the file size is too big that even PerfView has problem opening it or if the VM is having Disk Space issues on D drive or the site's temp folder then we must lower the timeout value